app/js/main.js

```diff
@@ -1,71 +1,67 @@
-import angular from 'angular';
+import angular from "angular";
```

Comment:
We use ESLint for `adage-server`, so double-quoted strings would trigger an error there. It appears that Cognoma uses JSHint and perhaps has no problem with them, so if you prefer that I'd say go with it!

Reply:
Yup, Cognoma's actually just using [prettier](https://jlongster.com/A-Prettier-Formatter) for JS formatting with the default settings 👍